filters/builtin/headerfilter.go

```diff
@@ -244,15 +252,43 @@ func valueFromContext(
 	}
 }
 
+func renderValueTemplate(template *eskip.Template, ctx filters.FilterContext) (string, bool) {
```

Comment:
The result value is "ok" style. Does it make sense to either have it a named result or a short function documentation?
Example:
```
 func renderValueTemplate(template *eskip.Template, ctx filters.FilterContext) (result string, ok bool) {
...
ok = !missing
return
```

Reply:
I have named return values, thanks for suggestion.